server/database.js

```diff
@@ -0,0 +1,12 @@
+require("dotenv").config();
+const Pool = require("pg").Pool;
+
+const pool = new Pool({
+   user: process.env.USER,
+   password: process.env.DB_PASSWORD,
+   database: process.env.DB_NAME,
+   host: process.env.HOST,
+   port: process.env.PORT,
```

Comment:
PostgreSQL has defaults for several of these. Do you think adding them as optional env values and then setting defaults here would be a good idea?

Reply:
In order for the dev to start up my project faster, I would say yes that it is a better idea to have optional env values. From what I know, PostgreSQL has default values for `user`, `host`, and `port`. So to have optional env values for those values would save some time.